renderer/styles/horizontal_and_vertical.css

```diff
@@ -8,7 +8,7 @@
 
 .mcq-content-container .question-content-container {
     height: 23.33333vh;
-    font-size: 3.9vh;
+    font-size: 14px;
```

Comment:
This should be in vh not in px value. It will break in different resolution devices.